internal/types/invoice.go

```diff
@@ -259,6 +259,9 @@ type InvoiceFilter struct {
 	// amount_remaining_gt filters invoices with an outstanding balance greater than the specified value
 	// Useful for finding invoices that still have significant unpaid amounts
 	AmountRemainingGt *decimal.Decimal `json:"amount_remaining_gt,omitempty" form:"amount_remaining_gt"`
+
+	// SkipLineItems if true, will not include line items in the response
+	SkipLineItems bool `json:"skip_line_items,omitempty" form:"skip_line_items"`
```

Comment:
This should ideally not be part of this PR and is already there on develop code
not sure why is git still showing as diff